docs/framework/guides/custom-editor-creator.md

```diff
@@ -1,460 +0,0 @@
----
```

Comment:
I decided to remove this guide. It is useful as it describes how to handle `Editor` classes and how to create your own custom editor implementation. But the guide was describing the multi-root editor and it is redundant at this point, might even be confusing. Since I don't have an idea for a different editor (nor time to think on it or write a new guide), I removed this one.

Also think that multi-root editor will solve a lot of custom needs.

Reply:
cc @apadol-cksource @Mgsy